W-21461047-update-http-connector-feedback-sis (#3168)
mulesoft/docs-connectors: http/1.11/modules/ROOT/pages/http-documentation.adoc

| Max Connections a| Number |  Maximum number of connections to open to the backend. HTTP requests are sent in parallel over multiple connections. Setting this value too high can impact latency and consume additional resources without increasing throughput. By default the number of connections is unlimited. |  -1 |
| Connection Idle Timeout a| Number |  The number of milliseconds that a connection can remain idle before it's closed. The value of this attribute is only used when persistent connections are enabled. |  30000 |
| Stream Response a| Boolean |  If this value is `true`, Mule streams received responses, meaning processing continues as soon as all headers are parsed and the body streamed as it arrives. When enabled, the response must be eventually read since depending on the configured buffer size it may not fit into memory and processing stops until space is available. |  false |

> | Response Buffer Size a| Number |  Size of the buffer that stores the HTTP response, in bytes. A value of `-1` (the default) means the buffer size is not limited; the connector uses a system-defined default. Specify a positive number to set an explicit buffer size in bytes. |  -1 |

Reviewer: ```suggestion
| Response Buffer Size a| Number |  Size of the buffer that stores the HTTP response, in bytes. A value of `-1` (the default) means the buffer size isn't limited and that the connector uses a system-defined default. Specify a positive number to set an explicit buffer size in bytes. |  -1 |
```
